blog/2024/10-26-media-segments-10.10.0/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Jellyfin Media Segments Explained 10.10.0
+description: What are Media Segments and how to use them in version 10.10.0
+authors: JPVenson
+slug: jellyfin-media-segments-10.10.0
+tags: [release]
+---
+
+{/* truncate */}
+
+## Media Segments Overview
+
+First lets start with the basics of what a Media Segment is.
+A Media Segment exists only within our database and defines a timespan within a movie or episode with a certain types.
```

Comment:
```suggestion
First lets start with the basics of what a media segment is. A media segment exists only within our database and defines a timespan within a movie or episode with a certain type.
```

The database part is too technical for average user, so I'd remove it.

```suggestion
First lets start with the basics of what a media segment is. A media segment defines a timespan within a movie or episode with a certain type.
```

Reply:
May also want to start with something like that it is a new feature in 10.10 and link to the release post there